Tribler/Main/Utility/GuiDBHandler.py

```diff
@@ -166,40 +162,33 @@ def wrapper():
         wrapper.__name__ = str(name)
 
         if not self.onSameThread(workerType) or delay:
+            task_name = uId
```

Comment:
The add_task will generate a task_name if name is None, I guess this can be removed.